python/lsst/ts/xml/data/sal_interfaces/MTHexapod/MTHexapod_Events.xml

```diff
@@ -411,4 +411,28 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>MTHexapod</Subsystem>
+    <EFDB_Topic>MTHexapod_logevent_highCurrent</EFDB_Topic>
```

Comment:
Are you sure you want to add this to the CSC? why not let the Watcher take care of this?